packages/data-point-service/examples/express-cache-stale-while-revalidate-instance-entity.js

```diff
@@ -45,7 +45,7 @@ function createService () {
         },
         params: {
           cache: {
-            ttl: '1m',
```

Comment:
this change I can take off if you guys want, but it was making it really hard to see if it was all working